packages/coinstac-ui/app/render/state/root-reducer.js

```diff
@@ -20,7 +20,7 @@ const rootReducer = combineReducers({
   loading,
   project,
   projects,
-  remoteResults,
+  resultsState,
```

Comment:
What would you think about naming this part of the state tree `results` instead of `resultsState`? Bit terser, and the keys are `localResults` and `remoteResults` so it shouldn't cause confusion (no `results.results` plucking).